src/main/org/testng/eclipse/launch/SuiteSelector2.java

```diff
@@ -30,7 +30,7 @@
   SuiteSelector2(TestNGMainTab callback, Composite comp) {
     super(callback, comp, LaunchType.SUITE, "TestNGMainTab.label.suiteTest",
         "CheckBoxTable.suites.title");
-    setTextEditable(false); // no manual edit
+    setTextEditable(true); // no manual edit
```

Comment:
yes, indeed, just checked in the changes, see the updated pull request:
https://github.com/cbeust/testng-eclipse/pull/130/files

thanks

  Thanks & Best Regards!

                  ///
                 (. .)
  --------ooO--(_)--Ooo--------
  |           Nick Tan           |
  ------------------------------------

On Sat, Apr 18, 2015 at 3:34 PM, Jean-Noël Rouvignac <
notifications@github.com> wrote:

> In src/main/org/testng/eclipse/launch/SuiteSelector2.java
> <https://github.com/cbeust/testng-eclipse/pull/130#discussion_r28642179>:
>
> > @@ -30,7 +30,7 @@
> >    SuiteSelector2(TestNGMainTab callback, Composite comp) {
> >      super(callback, comp, LaunchType.SUITE, "TestNGMainTab.label.suiteTest",
> >          "CheckBoxTable.suites.title");
> > -    setTextEditable(false); // no manual edit
> > +    setTextEditable(true); // no manual edit
>
> You should remove the associated comment - which should never have been
> there in the first place and is now outdated.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cbeust/testng-eclipse/pull/130/files#r28642179>.
>